docs/adding games.md

```diff
@@ -31,48 +38,94 @@ privilege can be lost, requiring the room to be moved to a new port
 * Keep an index for items received in order to resync. The ItemsReceived Packets are a single list with guaranteed 
 order.
 * Receive items that were sent to the player while they were not connected to the server
-  * The player being able to complete checks while offline and sending them when reconnecting is a good bonus, but not 
-strictly required
 * Send a status update packet alerting the server that the player has completed their goal
 
-Libraries for most modern languages and the spec for various packets can be found in the
-[network protocol](/docs/network%20protocol.md) API reference document.
+### Encouraged Features
+
+These are "nice to have" features for a client, but they are not strictly required. It is encouraged to add them 
+if possible.
+
+* The player being able to complete checks while offline and sending them when reconnecting
```

Comment:
I think there is a slight difference here between two concepts

1. If you're not connected, all randomization features should work correctly
2. If you're not connected and do a one-time in-game action that will send a check (e.g. opening a chest), and then you connect, the check should send instead of being lost permanently


I don't really agree with 1, but I agree with 2, in fact I would almost call 2 a requirement. I read this as 1 initially.